sys/include/ubjson.h

```diff
@@ -17,9 +17,9 @@
  */
 
 /**
- * @defgroup    sys_ubjson Universal Binary JSON
- * @ingroup     sys
- * @brief       A library to read and write UBJSON serialized data.
+ * @defgroup sys_ubjson  Universal Binary JSON deserializer
+ * @ingroup  sys_serialization
+ * @brief    Provides a BSON deserializer library to RIOT
```

Comment:
BSON?

Reply:
To clarify my question after a little bit of research: I don't think BSON and UBJSON are the same thing. [According to Wikipedia](https://en.wikipedia.org/wiki/UBJSON) UBJSON is aimed to be the successor of BSON, but that still wouldn't be the same thing.

Reply:
ok, I will differentiate them